services/core-web/src/components/Forms/projectSummaries/ProjectSummaryDocumentUpload.js

```diff
@@ -85,6 +85,7 @@ export class ProjectSummaryDocumentUpload extends Component {
             documents={this.props.initialValues?.documents?.reduce(
               (docs, doc) => [
                 {
+                  ...doc,
```

Comment:
I think that we've got some extraneous attributes here, lots of x: doc.x that should now be taken care of with ...doc. I think the key should already be in the doc, which would just leave the category to be specified.